src/sql/workbench/contrib/profiler/browser/profilerTableEditor.ts

```diff
@@ -53,7 +53,7 @@ export class ProfilerTableEditor extends EditorPane implements IProfilerControll
 	private _findCountChangeListener: IDisposable;
 	private _findState: FindReplaceState;
 	private _finder: FindWidget;
-	private _overlay: HTMLElement;
+	public _overlay: HTMLElement;
```

Comment:
nit: public props shouldn't start with _

Reply:
Updated it.